### DISCUSSION TITLE
Condon on multiprocess

### DISCUSSION THREAD

**Discussion by DanielGoldfarb** [exaloop/codon#535] [Q&A]
This is my first time using codon.  I have a python script using python's multiprocess module running 8 processes to keep my 8 cores busy.  It runs fine with python, but takes about 3.5 minutes to run.

With codon I get:
```text
$ codon run ssm_multiprocess.py
ssm_multiprocess.py:1:8-20: error: no module named 'multiprocess'
ssm_multiprocess.py:2:8-14: error: no module named 'pandas'
random.codon:263:13-16: error: name 'INF' is not defined
$
```

Any ideas what I can do to fix this issue?

**Comment by ypfmde** [comment]
The message indicates that you misspelled the name of the `multiprocessing` library. In fact
`from python import multiprocessing` works just fine for me. (But of course, I prefer `@par` from codon.)

**Comment by DanielGoldfarb** [comment]
There are two different multiprocessing libraries ... 

Take a look:

https://www.google.com/search?q=multiprocess+vs+multiprocessing+python

The failure occurs with both.   This appears maybe to be some kind of a path issue such that Condon can't find the library.

**Reply by elisbyberi** [reply]
It looks like you are trying to import the multiprocess library as a Codon module (file) rather than a Python library.

See https://docs.exaloop.io/codon/interoperability/python
Use `from python import multiprocess`

**Reply by DanielGoldfarb** [reply]
@elisbyberi  thanks.  I will try that.
